packages/patternfly-4/react-core/src/components/Accordion/Accordion.tsx

```diff
@@ -2,22 +2,44 @@ import * as React from 'react';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/Accordion/accordion';
 
+export const AccordionContext = React.createContext('h3');
+
+export enum HeadingLevel {
+  h1 = 'h1',
+  h2 = 'h2',
+  h3 = 'h3',
+  h4 = 'h4',
+  h5 = 'h5',
+  h6 = 'h6'
+}
+
+// declare global {
+//   namespace JSX {
+//     interface IntrinsicElements {
+//       'HeadingLevel': React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement>;
+//     }
+//   }
+// }
+
 export interface AccordionProps extends React.HTMLProps<HTMLDListElement> {
   /** Content rendered inside the Accordion  */
   children?: React.ReactNode;
   /** Additional classes added to the Accordion  */
   className?: string;
   /** Adds accessible text to the Accordion */
   'aria-label'?: string;
+  /** the heading level to use */
+  headingLevel?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
 }
 
 export const Accordion: React.FunctionComponent<AccordionProps> = ({
   children = null,
   className = '',
   'aria-label': ariaLabel = '',
+  headingLevel = 'h3',
   ...props
 }: AccordionProps) => (
   <dl className={css(styles.accordion, className)} aria-label={ariaLabel} {...props}>
-    {children}
+    <AccordionContext.Provider value={ headingLevel }>{children}</AccordionContext.Provider>
```

Comment:
I guess we still have eslint turned off, but I think when it's eventually turned back on it'll complain about the spacing here, I think you want to remove the spaces around `headingLevel` here (`value={headingLevel}`)

Reply:
Thank you! I resolved changes you mentioned above in this commit [https://github.com/patternfly/patternfly-react/pull/2273/commits/9db9ef3ff99503ea299d88585119f0b117745ad7](here)